veocean-data-farming/df-background.md

```diff
@@ -0,0 +1,55 @@
+
+
+## Active Work to Drive APY
+
+Data Farming is not a wholly passive activity. The name of the game is to drive Data Consume Volume (DCV). High APYs happen only when there is sufficiently high DCV. High DCV means publishing and consuming truly useful datasets (or algorithms).
+
+Thus, if you really want to max out your APY:
+- create & publish datasets (and make $ in selling them) — or work with people who can
+- consume the datasets (to make $) — or work with people who can
+- go stake on them, and finally claim the rewards.
+
+Driving DCV for publishing & consuming is your challenge. It will take real work. And then the reward is APY. It’s incentives all the way down:)
+
+## Reward Function
+
+The Reward Function (RF) governs how active rewards are allocated to stakers.
+
+Rewards are calculated as follows:
+1. Distribute OCEAN across each asset based on rank: highest-DCV asset gets most OCEAN, etc.
+1. For each asset and each veOCEAN holder:
+– If the holder is a publisher, 2x the effective stake
+– Baseline rewards = (% stake in asset) * (OCEAN for asset)
+– Bound rewards to the asset by 125% APY
+– Bound rewards by asset’s DCV * 0.1%. This prevents wash consume.
+
+You can find this code inside [calcrewards.py](https://github.com/oceanprotocol/df-py/blob/main/util/calcrewards.py) in the Ocean Protocol [df-py repo](https://github.com/oceanprotocol/df-py/)
+
+## Data Assets that Qualify for DF
+
+Data assets that have veOCEAN allocated towards them get DF rewards.
+
+The data asset may be of any type — dataset (for static URIs) or algorithm for Compute-to-Data. The data asset may be fixed price or free price. If fixed price, any token of exchange is alright (OCEAN, H2O, USDC, ..).
+
+To qualify for DF, a data asset must also:
+- Have been created by Ocean Smart contracts [deployed](https://github.com/oceanprotocol/contracts/blob/v4main/addresses/address.json) by OPF to [production networks](https://docs.oceanprotocol.com/core-concepts/networks)
+- Be visible on [Ocean Market](https://market.oceanprotocol.com/)
+- Can’t be in [purgatory](https://github.com/oceanprotocol/list-purgatory/blob/main/policies/README.md)
+
+## 3 Phases of Data Farming
+
+Data Farming has evolved over time and will continue to do so as the Emission Curve progresses. Below are important dates and parameters incurred during the evolution of the Data Farming program.
+
+**DF Alpha** - Counting starts Thu June 16. 10K OCEAN rewards are budgeted per week. Rewards are distributed at the end of every week, for the activity of the previous week. It runs 4 weeks. The aim is to test technology, learn, and onboard data publishers.
+
+**DF Beta** - Counting starts Thu July 14. Rewards are up to 100K OCEAN per week. It runs up to 20 weeks. The aim is to test the effect of larger incentives, learn, and refine the technology.
```

Comment:
You've written this in the present tense which is confusing as these are in the past. It's made worse because you haven't included the year, so next January people will be very confused. The phases that have ended should say something like 

"Ended on {Date}. 10K OCEAN rewards were budgeted per week. Rewards were distributed at the end of every week, for the activity of the previous week. It ran for 4 weeks. The aim was to test technology, learn, and onboard data publishers."

Reply:
Done. Please (see update and comments above)[https://github.com/oceanprotocol/docs/pull/1194#discussion_r1148250617].